src/main/java/software/aws/neptune/jdbc/Driver.java

```diff
@@ -39,15 +41,33 @@
     public static final String DRIVER_VERSION;
     public static final String APP_NAME_SUFFIX;
     public static final String APPLICATION_NAME;
+    private static final String PROPERTIES_PATH = "/project.properties";
+    private static final String MAJOR_VERSION_KEY = "driver.major.version";
+    private static final String MINOR_VERSION_KEY = "driver.minor.version";
+    private static final String VERSION_KEY = "driver.full.version";
     private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(Driver.class);
 
     static {
         APPLICATION_NAME = getApplicationName();
-        // TODO: Get driver version, suffix
-        DRIVER_MAJOR_VERSION = 1;
-        DRIVER_MINOR_VERSION = 1;
+        // TODO: suffix
```

Comment:
Why do we need TODIO for  APP_NAME_SUFFIX, can we remove  APP_NAME_SUFFIX as a whole?